services/catalog/service/jest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import { url as debugUrl } from 'inspector';
+
+//TODO: Move this to jest.config.js as `testTimeout` property when this issue is
+//resolved: https://github.com/facebook/jest/issues/9696
+if (debugUrl() !== undefined) {
+  jest.setTimeout(60000); // wait for longer when debugging a test
```

Comment:
If you are already on it: I would increase the timeout in debug scenario to a lot longer. Like 10 or 15 minutes would also not hurt I guess if you need to debug and then research a bit.